CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ endif()
 option(BUILD_SOX "Build libsox statically" ON)
 option(BUILD_KALDI "Build kaldi statically" ON)
 option(BUILD_RNNT "Enable RNN transducer" ON)
+option(BUILD_KENLM "Build KenLM statically" ON)
```

Comment:
do we want this to be supported by default?

Reply:
Yes, since we intend to include this in the next release with prototype. In the past, when the release engineering team manually released a binary for M1 mac, they did not know about BUILD_SOX switch, and it caused additional issue.
So I think it's safer to enable by default. Plus, this way if something is wrong, we will receive issue report from users building from source.